account_payment_return/models/payment_return.py

```diff
@@ -228,8 +227,10 @@ def action_confirm(self):
                 # returned_moves: debit on customer account (from invoice move)
                 returned_moves = move_line.matched_debit_ids.mapped(
                     'debit_move_id')
+                returned_moves.mapped('invoice_id')._payment_returned(
```

Comment:
Can this return more than one invoice and thus give singleton error?

Reply:
This may return more than one invoice when a payment reconciled with several invoices is returned.
`_payment_returned` is api.multi, it should not be problematic.